lib/network/encoding.go

```diff
@@ -66,6 +68,24 @@ func TypeFromData(msg ProtocolMessage) uuid.UUID {
 	return msgType
 }
 
+// TypeToUUID Converts MsgType to uuid
+func TypeToUUID(msg ProtocolMessage) uuid.UUID {
+	val := reflect.ValueOf(msg)
+	if val.Kind() == reflect.Ptr {
+		val = val.Elem()
+	}
+	url := "https://dedis.epfl.ch/protocolType/" + val.Type().String()
```

Comment:
I think we don't even need that actually. Using NewV5 (line just below), we already have to provide in which *namespace* we are, so NamespaceURL + MsgType is quite ... UUIdentifiable!